tests/spec/unit/create.spec.js

```diff
@@ -59,7 +59,15 @@ function createAndValidateProjectDirName (projectname, projectid) {
     const _fs = create.__get__('fs');
     create.__set__('fs', {
         ensureDirSync: _fs.ensureDirSync,
-        existsSync: path => !(path === projectPath),
+        existsSync: path => {
+            if (path === projectPath) {
```

Comment:
Just a nitpick, but you could join if and else if blocks together with OR operator. Or it is done this way intentionally?

Reply:
I did it intentionally but changed it.